test/utils/test-utils.ts

```diff
@@ -0,0 +1,146 @@
+import {Provider} from "@ethersproject/providers";
+import {parseEther} from "@ethersproject/units";
+import {SignerWithAddress} from "@nomiclabs/hardhat-ethers/signers";
+import {BigNumber, Signer} from "ethers";
+import hre from "hardhat";
+import {ethers} from "hardhat";
+import erc20ABI from "../../abi/erc20.json";
+import {IWidoRouter} from "../../typechain";
+
+const whaleAddress: {[key: string]: string} = {
+  // Mainnet
```

Comment:
Can we remove most of it and only keep USDC for mainnet?

Reply:
I kept WETH, which we use, and USDC on a couple of networks.